tests/test_cases/test_array/test_array.py

```diff
@@ -54,93 +70,93 @@ async def test_read_write(dut):
     await Timer(10, "ns")
 
     tlog.info("Checking Generics/Parameters:")
-    _check_logic(tlog, dut.param_logic    , 1)
+    _check_logic(tlog, dut.param_logic, 1)
     _check_logic(tlog, dut.param_logic_vec, 0xDA)
 
     if cocotb.LANGUAGE in ["vhdl"]:
         _check_int(tlog, dut.param_bool, 1)
-        _check_int(tlog, dut.param_int , 6)
+        _check_int(tlog, dut.param_int, 6)
         _check_real(tlog, dut.param_real, 3.14)
-        _check_int(tlog, dut.param_char, ord('p'))
-        _check_str(tlog, dut.param_str , b"ARRAYMOD")
+        _check_int(tlog, dut.param_char, ord("p"))
+        _check_str(tlog, dut.param_str, b"ARRAYMOD")
 
         if not cocotb.SIM_NAME.lower().startswith("riviera"):
-            _check_logic(tlog, dut.param_rec.a        , 0)
```

Comment:
I suppose this file has some useful manual alignment, but I don't have strong feelings about it.